### DISCUSSION TITLE
How to use DNS rewrite to use upstream record for TXT

### DISCUSSION THREAD

**Discussion by jon-is-me** [AdguardTeam/AdGuardHome#5932] [Q&A]
I have dns rewrite for `*.mydomain.com` to 192.168.1.123. I use acme.sh / other cert resolvers with DNS challenge in order to generate a cert. during this process it’ll create a TXT record for `_acme-challenge.mydomain.com`. The acme script can’t find the TXT record because AdGuard is doing a rewrite.

Has anyone gotten this to work?

**Comment by Blu3wolf** [comment]
Well at least Im not the only one facing this issue! I dont suppose you figured out a workaround?

Edit: ~~I figured out a workaround, using the Custom Filtering rules.~~

`||sample.domain.tld^$dnstype=A,dnsrewrite=NOERROR;A;192.168.50.2`

~~With the above, A records from upstream are rewritten to 192.168.50.2, but other records are unaffected.~~

~~Im not sure what happened, but this solution isnt working for me.~~

Okay, it does work after all. Turns out my issue was upstream DNS propagation, not a fault of the filter rule.
